### PR TITLE
feat: bootstrap nav

### PR DESCRIPTION
**Pull request by Abdur-rahmaanJ**

### CODE REVIEW

**Comment by Abdur-rahmaanJ** [comment]
pr_bs

**Comment by Abdur-rahmaanJ** [comment]
Just a demo, which bootstrap to use??

ng or bootsrap or ...

File: src/styles.css [hunk 1]
```diff
@@ -0,0 +1 @@
+@import "~bootstrap/dist/css/bootstrap.css"
```
Comment (kherin) [inline src/styles.css]: Missing newline at end of file

**Review by kherin — CHANGES_REQUESTED** [review]
Overall looks good to me 👍

**Pull request merged**